aries-site/src/examples/components/avatar/AvatarDarkModeExample.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Avatar, Box } from 'grommet';
+
+export const AvatarDarkModeExample = () => (
+  <Box
+    width="medium"
+    height='small'
```

Comment:
doesn't look like linting happened correctly here. There's inconsistencies with the " vs ' quotations

Reply:
fixed